docs/getting-started/sdk/secrets-manager/integrations/kubernetes.mdx

```diff
@@ -0,0 +1,301 @@
+
+:::caution
+
+The Dev Container will not work correctly if you are using a container engine other than Docker.
+
+:::
+
+</TabItem>
+<TabItem value="direct" label="Direct Setup">
+
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Visual Studio Code Go Extension](https://marketplace.visualstudio.com/items?itemName=golang.go)
+- [Go](https://go.dev/dl/) version 1.20 or 1.21
+- [Operator-SDK](https://sdk.operatorframework.io/docs/installation/#install-from-github-release)
+- [musl-gcc](https://wiki.musl-libc.org/getting-started.html)
+- [Make](https://www.gnu.org/software/make/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [Docker](https://www.docker.com/) or [Podman](https://podman.io/) or another container engine
+- A [Kind Cluster](https://kind.sigs.k8s.io/docs/user/quick-start/) or other Kubernetes cluster with
+  Kubectl pointed to it as the current context for local development.
+
+</TabItem>
+</Tabs>
+
+You will also need:
+
+- A
+  [Bitwarden Organization with Secrets Manager](https://bitwarden.com/help/sign-up-for-secrets-manager/).
+  You will need the organization ID GUID for your organization.
+- An [access token](https://bitwarden.com/help/access-tokens/) for a Secrets Manager service account
+  tied to the projects you want to pull.
+
+## Setup and configuration
+
+Clone the repository:
+
+```bash
+git clone https://github.com/bitwarden/sm-kubernetes.git
+```
+
+Open Visual Studio Code to the
+[src/sm-operator](https://github.com/bitwarden/sm-kubernetes/tree/main/src/sm-operator) subfolder.
+Opening here will allow all of the Dev Container, Go language extensions, tasks, and launch settings
+to work correctly.
+
+<Tabs>
+<TabItem value="dev-container" label="Dev Container" default>
+
+Dev Container setup is automated. This will create a Kind cluster and setup all necessary software
+and SDK binaries.
+
+- Press `F1` to open the command palette
+- Start typing `Dev Containers: Reopen in Container`
+
+</TabItem>
+<TabItem value="direct" label="Direct Setup">
+
+:::info
+
+Please note that the Visual Studio Code debugger for Go does not work correctly if you open your
+workspace via a symlink anywhere in the path. For debugging to work, you should open it from the
+full path of the repository.
+
+:::
+
+- Install all requirements listed in the Direct Setup requirements
+- Create a Kind cluster using `kind create cluster`
+- Run `make setup` to download the appropriate Secrets Manager SDK binaries
+
+</TabItem>
+</Tabs>
+
+### Configuration settings
+
+A `.env` file will be created under
+[src/sm-operator](https://github.com/bitwarden/sm-kubernetes/tree/main/src/sm-operator) once the Dev
+Container is created or `make setup` has been run. The following environment variable settings can
+be updated to change the behavior of the operator:
+
+- **BW_API_URL** - Sets the Bitwarden API URL that the Secrets Manager SDK uses. This is useful for
```

Comment:
⛏️ I usually like to end multi-sentence bullets with a period.

Reply:
I do if it's a complete sentence or has multiple sentences.  I usually do not if it is not a complete sentence.

Reply:
Which this one is... :-D

Reply:
Added.